backend/.eslintrc.js

```diff
@@ -1,17 +1,27 @@
 module.exports = {
   parser: "@typescript-eslint/parser",
   extends: [
+    "plugin:react/recommended",
```

Comment:
Why are we adding react linting to our backend?

Reply:
I'm not sure why that got added, I don't remember intentionally changing the eslint files at all.